docs/docs.go

```diff
@@ -0,0 +1,268 @@
+                    "type": "boolean"
+                },
+                "map_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "map_name": {
+                    "type": "string"
+                },
+                "memory": {
+                    "type": "integer"
+                },
+                "monitor_maps": {
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/models.L3afDNFMetricsMap"
+                    }
+                },
+                "name": {
+                    "type": "string"
+                },
+                "prog_type": {
+                    "type": "string"
+                },
+                "rules": {
+                    "type": "string"
+                },
+                "rules_file": {
+                    "type": "string"
+                },
+                "seq_id": {
+                    "type": "integer"
+                },
+                "start_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "status_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "stop_args": {
+                    "$ref": "#/definitions/models.L3afDNFArgs"
+                },
+                "user_program_daemon": {
+                    "type": "boolean"
+                },
+                "version": {
+                    "type": "string"
+                }
+            }
+        },
+        "models.BPFPrograms": {
+            "type": "object",
+            "properties": {
+                "tc_egress": {
+                    "description": "list of tc egress bpf programs",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/models.BPFProgram"
+                    }
+                },
+                "tc_ingress": {
+                    "description": "list of tc ingress bpf programs",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/models.BPFProgram"
+                    }
+                },
+                "xdp_ingress": {
+                    "description": "list of xdp bpf programs",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/models.BPFProgram"
+                    }
+                }
+            }
+        },
+        "models.L3afBPFPrograms": {
+            "type": "object",
+            "properties": {
+                "bpf_programs": {
+                    "description": "list of bpf programs",
+                    "$ref": "#/definitions/models.BPFPrograms"
+                },
+                "host_name": {
+                    "description": "host name or pod name",
+                    "type": "string"
+                },
+                "iface": {
```

Comment:
can you expand "iface" and just say "interface"?

Reply:
Needs code change, will do it later.